webcompat/form.py

```diff
@@ -140,7 +140,10 @@ def normalize_url(url):
         return None
     url = url.strip()
     parsed = urlparse.urlparse(url)
-
+    # Handle the case when URL has the form http://https://example.com
+    if parsed.netloc in ['http:', 'https:'] and parsed.path.startswith('//'):
+        url = url.split('//', 1)[1]
```

Comment:
So we take the "inner" value, right?

Reply:
@miketaylr yes. I assumed that if people are botching this thing is because they had to an already existing string and not insert in the middle of it… ahah. We might get surprises. People are creative.

Reply:
```python
>>> url = 'http://https://bad.example.com'
>>> # Splitting on all doubles '//'
>>> url.split('//')
['http:', 'https:', 'bad.example.com']
>>> # Splitting on the first instance from the left (for the right rsplit)
>>> url.split('//', 1)
['http:', 'https://bad.example.com']
>>> # Takes the second item on the list only
>>> url.split('//', 1)[1]
'https://bad.example.com'
```

Another assumption: This will happen only once. I don't expect people to do multiple `http://http://http://` if we are wrong, we can still go recursive. :)